style.css

```diff
@@ -1,4 +1,74 @@
 body {
-background-color: #232323
+  color: #ffffff;
+  font-family: 'Ubuntu', sans-serif;
+  font-size: 15px;
+  width: auto;
+  background: url(nieckovsky_image_main.jpg);
+  background-color: #111415;
+  background-size: 100%;
+  background-position: center;
+  background-attachment: fixed;
+  background-repeat: no-repeat;
+}
+
+header {
+  width: 100%;
+  height: 50px;
+}
+
+nav {
+  display: inline-block;
+  width: 100%;
+  position: relative;
+  margin-top: auto;
+  margin-bottom: auto;
+}
+
+span {
```

Comment:
While I admire that you wrote a semantic markup where an image actually has a note on it for screen readers, I don't think `span` elem is our way to go. In order to make this span act like an image we got 3 `background-` attributes, `font-size: 0`, `line-height: 0` and other unnecessary hacks. I think we're good just with `<img src="nieckovsky_logo_top.png" alt="Nieckovsky portfolio"/>` and some minimalistic CSS rules to make it happen like `width`, `height`, `margin-left` and `vertical-align`. Not only do we shrink the CSS, but also improve HTML markup.

Reply:
Thanks, changed span for img.